conf.d/Makefile

```diff
@@ -22,6 +22,9 @@ CONF-NetBSD=	moused rarpd savecore
 
 include ${MK}/scripts.mk
 
+modules: modules${SFX}
+	cp $@${SFX} $@
```

Comment:
Eww... See the next target for a much better way to do this (we really should find a way to make these out of generic targets, though)